__tests__/app.test.js

```diff
@@ -67,7 +67,26 @@ describe("GET /api/articles/:article_id", () => {
             author: "butter_bridge",
             body: "I find this existence challenging",
             created_at: "2020-07-09T20:11:00.000Z",
-            votes: 100
+            votes: 100,
+            comment_count: "11"
```

Comment:
You could recast the comment_count as an integer if you want - this would save us having to deal with this later on